src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -44,6 +44,20 @@ def makeList(stringList):
     raise WMSpecFactoryException("Can't convert to list %s" % stringList)
 
 
+def makeNonEmptyList(stringList):
```

Comment:
Definitely no dependencies from Utils onto something else. The idea with Utils is that *any* python project might benefit just by grabbing it.

What would it take to just move both make*List functions to Utils?